Volume/View Models/FlyersViewModel.swift

```diff
@@ -70,6 +75,18 @@ extension FlyersView {
             await fetchContent()
         }
         
+        func fetchUpcoming() async {
+            // TODO: Fetch flyers under "Upcoming"
+            
+            // Get flyers that is later than now and matches the current category
+            if selectedCategory == .all {
+                upcomingFlyers = FlyerDummyData.flyers.filter { $0.date.start > Date() }
+            } else {
+                upcomingFlyers = FlyerDummyData.flyers.filter { $0.date.start > Date() && $0.organizations[0].contentType == selectedCategory }
```

Comment:
this filtering logic might be wrong if a flyer is in multiple organizations because you only check `organizations[0].contentType`